src/css/_docs.scss

```diff
@@ -481,12 +482,25 @@
     &:hover {
       opacity: 1;
     }
+    .progress {
+      z-index: 10;
+      position: absolute;
+      top: -1px;
+      left: 0;
+      height: 1px;
+      background-color: $c-grey-lightest;
+      opacity: 0;
+      transition: width 300ms;
+    }
   }
   &-active {
     //Apologies but can only make work with !important
-    border-top: 1px solid $c-grey-lightest !important;
+    border-top: 1px solid $c-grey-light !important;
     opacity: 1;
     transition: all 360ms;
+    .progress {
+      opacity: 1 !important;
```

Comment:
`!important` are used as a last resort css 😄 can we add a comment explaining why the `!important` is used?